graph/patterns/change-tracking.md

```diff
@@ -0,0 +1,161 @@
+# Change tracking
+
+Microsoft Graph API Design Pattern
+
+*The change tracking pattern provides the ability for API consumers to request changes in data from Microsoft Graph without having to re-read data that has not changed.*
+
+
+## Problem
+
+API consumers require an efficient way to acquire changes to data in the Microsoft Graph, for example to synchronize an external store or to drive a change-centric business process.
+
+## Solution
+
+API designers can enable the change tracking (delta) capability on a resource in the Microsoft Graph (typically on an entity collection or a parent resource) by declaring a delta function on that resource.
```

Comment:
@garethj-msft   IMO we need to clarify that change tracking is enabled by declaring a delta function and applying ChangeTracking annotation to the targeted resource.

Reply:
We're also about to release support for change tracking on a function - but I guess we can update the doc once that's officially released.

Reply:
Add ChangeTracking annotation